src/utils/parser/index.mjs

```diff
@@ -103,6 +104,14 @@ export const transformTypeToReferenceLink = type => {
       return DOC_TYPES_MAPPING_NODE_MODULES[lookupPiece];
     }
 
+    // Transform Node.js types like 'vm.Something'.
+    if (lookupPiece.indexOf('.') >= 0) {
+      const [mod, ...pieces] = lookupPiece.split('.');
+      const isClass = pieces.at(-1).match(/^[A-Z][a-z]/);
```

Comment:
Is this regex just to match latin characters? This regex can be simplified, if that's the case you can simplify it with `/^[A-Za-z]/`

Also I wonder what is more expensive... `.at(-1).match(/^.../)` or `.match(/[A-Za-z]$/)` (match at end of string)

Reply:
Or wait, pieces is not a string, an array, so `.at()` is not a character lookup, but looking-up the last item of the array?

I wonder if this works:

```ts
const [mod,. ..._unused, last] =
```
Haven't ever tested so I doubt it does haha, but what if? (I forget if spread will grab all entries)

Anyways, the current logic looks good, but maybe regex can be the one I shared at the beginning of the msg? 👀

Reply:
It's matching an Uppercase character followed by a lowercase one

Reply:
Sorry, when I say latin, I mean alphabetic/words, whatever the correct terminology is. And true, apologies, again, half asleep lol